tutorials/mobile_automation_setup.rst

```diff
@@ -1,5 +1,5 @@
-How to automate nightly Google Play deployments
-===============================================
+Automate Google Play Deployments
```

Comment:
Is this a Tutorial or HOWTO? It reads more task-oriented than learning-oriented to me.